index.js

```diff
@@ -169,10 +176,61 @@ module.exports = (robot) => {
     authorize(JSON.parse(clientSecret), checkClaSheet);
   };
 
+  var checkMergeConflicts = async function(context) {
+    var mergeConflictLabel = ['PR: don\'t merge - HAS MERGE CONFLICTS'];
+    pullRequestsPromiseObj = await context.github.pullRequests.getAll(
+      context.repo());
+
+    arrayOfOpenPullRequests = pullRequestsPromiseObj.data;
+    var hasMergeConflictLabel;
+    for (var indexOfPullRequest in arrayOfOpenPullRequests) {
+      pullRequestNumber = await arrayOfOpenPullRequests[
+        indexOfPullRequest].number;
+      pullRequestDetailsPromiseObj = await context.github.pullRequests.get(

```

Comment:
You might want to convert this to an async (within an await) in a for-loop outside that that you can grab all of the data at once.

Reply:
Sorry but each of the `context.github.` statements call a GitHub API which returns a promise. All promises need to be resolved before moving on to the next statement.